downloader.py

```diff
@@ -161,9 +161,16 @@ def main():
                 print("{}".format(el['msg']))
 
             filename = os.path.join(folder_name, 'Video_' + str(week) + '.mp4')
-            print('Downloading file ', filename)
-            urllib.request.urlretrieve(el['url'], filename)
-            print('Done')
+            if not os.path.isfile(filename):
+                try:
+                    print('Downloading file ', filename)
+                    urllib.request.urlretrieve(el['url'], filename)
+                    print('Done')
+                except:
+                    os.remove(filename)
```

Comment:
It's better to catch specific exceptions, not all of them (`except Exception as e:`)
There is difference between those two construction, you now catch `KeyboardInterrupt` exception, which might have sense, in case you want to cancel script execution with ctrl-c and and remove partially downloaded file.
[difference-between-except-and-except-exception-as-e-in-python](http://stackoverflow.com/questions/18982610/difference-between-except-and-except-exception-as-e-in-python)
But, you removing filename right after exception it thrown, there is no guarantee that fill will exists at this point.
For example if there is no file, and you trying to download it, but at this point connection is lost you will receive:
```urllib.error.URLError: <urlopen error [Errno 64] Host is down>```
No file will be created and `os.remove` will throw
```FileNotFoundError: [Errno 2] No such file or directory: ...```